fineract-provider/src/main/java/org/apache/fineract/notification/service/NotificationDomainServiceImpl.java

```diff
@@ -552,7 +557,7 @@ private void buildNotification(String permission, String objectType, Long object
 		
 		String tenantIdentifier = ThreadLocalContextUtil.getTenant().getTenantIdentifier();
 		Queue queue = new ActiveMQQueue("NotificationQueue");
-		List<Long> userIds = retrieveUsersWithSpecificPermission(permission);
+		List<Long> userIds = retrieveSubscribers(officeId, permission);
```

Comment:
At the time of deployment of this code, we can assume that there will be no topics that have been created for existing offices and users. What is your plan to populate the topics table for existing offices and users?